web/themes/new_weather_theme/assets/sass/components/hourly-table.scss

```diff
@@ -15,7 +28,7 @@
   td, th {
     padding-left: 0;
     padding-right: units(3);
-    vertical-align: text-top;
+    vertical-align: top;
```

Comment:
Does this fix both the arrow size *and* the padding? (I ask because the other changes all appear to be related to the scroll buttons.)

Reply:
This line only fixes the wonky space and double border in the table cell on iPhone browsers. The arrow sizing is fixed by setting explicit padding on the arrow buttons. 

<img width="646" alt="screenshot of bad spacing in table on iphone" src="https://github.com/weather-gov/weather.gov/assets/150970973/f89ecd6b-80bf-4d09-bff7-65e4a26e3fe5">

Reply:
Oops, I was looking at the wrong arrows. Got it!